stories/components/heroBanner/index.js

```diff
@@ -0,0 +1,9 @@
+import react from 'react';
+import { storiesOf } from '@storybook/react';
+import HeroBanner from 'shared/components/heroBanner/heroBanner';
+
+import jamesBondJpg from '../../asset/james-bond.jpg';
+
+storiesOf('HeroBanner', module).add('Default', () => (
+  <HeroBanner imageSrc={jamesBondJpg} title="Bond" subtitle="James Bond" />
+));
```

Comment:
bless your soul for keeping Storybook up-to-date.

Reply:
I'm trying. I have no idea what I'm doing and it's not working, but I'm trying lol